atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/TimestampCommand.java

```diff
@@ -31,17 +35,35 @@
     		description = "Get the current immutable timestamp")
     private boolean immutable;
     
+    @Option(name = {"-d", "--date-time"},
+            description = "Return the earliest real point in time at which the immutable timestamp" +
```

Comment:
And with async puncher I think what we're retrieving with your logic here is "approximate wallclock time"